File: C#/백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.cs
using System;

namespace q{
    class w{
        static void Main(){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int min = 101;
                int sum = 0;
                for (int j=0;j<7;j++){
                    if (aa[j]%2==0){
                        min=min<aa[j]?min:aa[j];
                        sum+=aa[j];
                    }
                }
                Console.WriteLine($"{sum} {min}");
            }
        }
    }
}

File: Java/백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i=0;i<t;i++){
            int sum=0;
            int min=101;
            for (int j=0;j<7;j++){
                int n = in.nextInt();
                if(n%2==0){
                    sum+=n;
                    min = min<n?min:n;
                }
            }
            System.out.printf("%d %d\n",sum,min);
        }
    }
}